server/scheduler_boltdb.go

```diff
@@ -0,0 +1,291 @@
+package server
```

Comment:
I can't tell what changed here because the file was renamed. Can you change the name back?

Reply:
Sorry for the bad diff. You can see the changes by running: 

`git diff master:server/scheduler_service.go split-scheduler-service:server/scheduler_boltdb.go`